windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -28,6 +28,16 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-7-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 7.0.1
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.0.1 were released in https://access.redhat.com/errata/RHBA-2023:0748[RHBA-2023:0748].
+
```

Comment:
The above link is not yet active, until the [errata](https://errata.devel.redhat.com/advisory/109164) is published.